.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
+
+matrix:
+  allow_failures:
+   - php: nightly
+  fast_finish: true
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
```

Comment:
Add before_install: with - travis_retry composer self-update so that composer is up-to-date